packages/core/test/test_util/circular_list.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+exports.getCircularList = function getCircularList(arr) {
```

Comment:
I read somewhere that, for readability purposes, using the actual object/function as part of a module is better.
It was actually associated to ES6 modules, but the same applies here.
The basic idea is that whoever imports that function doesn't have to rename it to something else, even though in most cases I have seen, people tend to use the same name anyways.
However, by exporting the named object as part of the module and not the module itself, makes it 100% guaranteed that the object will be imported with its own name at all times.

For better illustration in both cases (ES6 modules and CommonJS):

```javascript
// inside of my-module, the function is called somethingElse()
import * as myFunction from './my-module'
```
against:

```javascript
// https://i.kym-cdn.com/entries/icons/facebook/000/020/147/drake.jpg
import somethingElse from './my-module'
```

And in our case, with CommonJS

```javascript
// inside of my-module, the function is called somethingElse()
const myFunction = require('./my-module')
```
against:

```javascript
// https://i.kym-cdn.com/entries/icons/facebook/000/020/147/drake.jpg
const { somethingElse } = require('./my-module')
```

I have seen in our code base that the "export as module" is largely used. I won't make a fuss about this, so if you want to keep the current style, I will gladly change it.
Just wanted to point out why I did it the way I did.

Reply:
Hm, good point. Never thought about it that way. I just tend to favor the other style I mentioned because it looks kinda neat for single purpose modules. But I don't have any strong opinion about it whatsoever. Both styles are present in the code base so by all means, there is no need to change this module here.